doc/lib_list_dialog.md

```diff
@@ -4,8 +4,8 @@
 
 This library helps when you want the user to choose a value from a long list.
 
-dependencies:
-  * lib_menu
+Dependencies:
+  * [lib_menu.ks](https://github.com/KSP-KOS/KSLib/blob/master/library/lib_menu.ks)
 
```

Comment:
Should `lib_gui_box.ks` be listed as a dependency, since `lib_menu.ks` depends on it?

Reply:
I don't know what the standard for that is, I could it wouldn't be to hard but if I do I should also list spec_char as well as as lib_gui_box supposedly depends on that.
On the other hand keeping that kind of recursive dependency information up to date by hand could get old fast and it would be a lot easier to miss something

Reply:
Yeah, that is true. Let's leave it to direct dependencies only.